packages/http/src/util.js

```diff
@@ -99,7 +99,10 @@ export function request(method, path, params, callback = s => s) {
 
     const options = {
       ...resolvedParams,
-      headers,
+      headers: {
```

Comment:
Worried about this fix. Isn't this going to break form uploads, because we're setting the header to `JSON` when actually we're sending `multi-part` form?